chain/beacon/node_test.go

```diff
@@ -233,6 +233,11 @@ func (b *BeaconTest) CreateNode(t *testing.T, i int) {
 			}
 		}
 	}
+	t.Cleanup(func() {
```

Comment:
Had a quick look online but couldn't see anything - if a caller calls `t.Cleanup` in their enclosing test, will it overwrite this closure or run in addition to it?

Reply:
Just found the answer myself:
```Cleanup registers a function to be called when the test (or subtest) and all its subtests complete. Cleanup functions will be called in last added, first called order. ```
noice